packages/edit-site/src/components/revisions/index.js

```diff
@@ -25,6 +25,7 @@ const {
 	ExperimentalBlockEditorProvider,
 	GlobalStylesContext,
 	useGlobalStylesOutputWithConfig,
+	__unstableBlockStyleVariationOverridesWithConfig,
```

Comment:
Why is this `__unstable`? It's already private, right?

Reply:
There was a [bit of discussion](https://github.com/WordPress/gutenberg/pull/62768#discussion_r1654254618) around it. 

To answer the "why", to advertise without ambiguity that, despite being private, it's unstable.

I don't think it does any harm, but could easily be communicated in comments. I can rename it next week.